### DISCUSSION TITLE
Getting Deepgram Error 429. Even for normal request and not even hitting the rate limit

### DISCUSSION THREAD

**Discussion by rkj412** [deepgram/community#940] [General help] [answered]
HI Team,

I am using pay as you go plan  and even I am not hitting the rate limit of 100. Every request is showing 429 error. Can you please suggest where I am going wrong.
![image](https://github.com/user-attachments/assets/e0e989a2-3b10-44f5-8dbc-c5b481a4ced9)

URL :

/v1/listen?encoding=linear16&sample_rate=8000&model=nova-2-phonecall&punctuate=true&interim_results=true&endpointing=300&utterance_end_ms=1000&smart_format=true&vad_events=true&channel=1

Request IDs: 
203c09ac-839a-43d5-aed2-d39733889103,
d778007a-fccb-4039-80a4-8683358a15a0,
f2693ffe-0393-4c54-bfca-bb6e06749dd5
501e680f-ec37-481c-9207-3e181d595dc4

**Comment by jjmaldonis** [comment, marked as answer]
I replied in Discord, see [here](https://discord.com/channels/1108042150941294664/1290630262710865983/1290680583847940156).

> At that time you had 100 streams open with Deepgram which is why the 429 was returned. It's possible those streams were being kept open with KeepAlive messages.

The discord thread contains a usage graph over that timespan.
